tdp/cli/commands/resume.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+from pathlib import Path
+
+import click
+
+from tdp.cli.session import get_session_class
+from tdp.cli.utils import check_services_cleanliness, collection_paths_to_collections
+from tdp.cli.commands.queries import get_deployment, get_last_deployment
+from tdp.core.dag import Dag
+from tdp.core.models.state_enum import StateEnum
+from tdp.core.runner import AnsibleExecutor, DeploymentPlan, DeploymentRunner
+from tdp.core.variables import ClusterVariables
+
+
+@click.command(short_help="Resume a TDP deployment")
+@click.option(
+    "--database-dsn",
+    envvar="TDP_DATABASE_DSN",
+    required=True,
+    type=str,
+    help=(
+        "Database Data Source Name, in sqlalchemy driver form "
+        "example: sqlite:////data/tdp.db or sqlite+pysqlite:////data/tdp.db. "
+        "You might need to install the relevant driver to your installation (such "
+        "as psycopg2 for postgresql)"
+    ),
+)
+@click.option(
+    "--collection-path",
+    "collections",
+    envvar="TDP_COLLECTION_PATH",
+    required=True,
+    callback=collection_paths_to_collections,  # transforms into Collections object
+    help=f"List of paths separated by your os' path separator ({os.pathsep})",
+)
+@click.option(
+    "--run-directory",
+    envvar="TDP_RUN_DIRECTORY",
+    type=Path,
+    help="Working directory where the executor is launched (`ansible-playbook` for Ansible)",
+    required=True,
+)
+@click.option(
+    "--vars",
+    envvar="TDP_VARS",
+    required=True,
+    type=click.Path(resolve_path=True, path_type=Path),
+    help="Path to the tdp vars",
+)
+@click.option("--dry", is_flag=True, help="Execute dag without running any operation")
+@click.argument("id", nargs=1, required=False, default=-1)
```

Comment:
are `nargs` and `default` required ? if you put `("id", required=False)`, then `id` should be None when no id provided no ?

Reply:
Yes it works, done in the latest commit